web/src/app/administration/components/classification-details/classification-details.component.spec.ts

```diff
@@ -0,0 +1,359 @@
+import { Component, DebugElement, Input } from '@angular/core';
+import { ClassificationsService } from '../../../shared/services/classifications/classifications.service';
+import { Observable, of } from 'rxjs';
+import { ClassificationCategoriesService } from '../../../shared/services/classification-categories/classification-categories.service';
+import { DomainService } from '../../../shared/services/domain/domain.service';
+import { ImportExportService } from '../../services/import-export.service';
+import { async, ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { Actions, NgxsModule, ofActionDispatched, Store } from '@ngxs/store';
+import { ClassificationState } from '../../../shared/store/classification-store/classification.state';
+import { EngineConfigurationState } from '../../../shared/store/engine-configuration-store/engine-configuration.state';
+import { classificationStateMock, engineConfigurationMock } from '../../../shared/store/mock-data/mock-store';
+import { ClassificationDetailsComponent } from './classification-details.component';
+import { FormsModule } from '@angular/forms';
+import { RequestInProgressService } from '../../../shared/services/request-in-progress/request-in-progress.service';
+import { FormsValidatorService } from '../../../shared/services/forms-validator/forms-validator.service';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatDialogModule } from '@angular/material/dialog';
+import { NumberPickerComponent } from '../../../shared/components/number-picker/number-picker.component';
+import { NotificationService } from '../../../shared/services/notifications/notification.service';
+import {
+  CopyClassification,
+  RemoveSelectedClassification,
+  RestoreSelectedClassification,
+  SaveCreatedClassification,
+  SaveModifiedClassification
+} from '../../../shared/store/classification-store/classification.actions';
+
+@Component({ selector: 'taskana-shared-number-picker', template: '' })
+class NumberPickerStub {
```

Comment:
You're right, I removed the stub. I think we replaced the stub by the component because of the error "No value accessor for form control with name: ... "